zerver/lib/test_helpers.py

```diff
@@ -90,6 +91,45 @@ def stub_event_queue_user_events(
             yield
 
 
+class activate_push_notification_service(override_settings):  # noqa: N801
```

Comment:
Just highlighting this helper as the main noteworthy thing in terms of the changes to test. this does all the work to keep the actual overrides done by tests simple like before

Reply:
Seems like a great abstraction!